thirdeye-server/src/main/java/ai/startree/thirdeye/auth/ResourceIdentifier.java

```diff
@@ -15,37 +15,50 @@
 
 import ai.startree.thirdeye.spi.datalayer.dto.AbstractDTO;
 import ai.startree.thirdeye.spi.datalayer.dto.AlertDTO;
+import ai.startree.thirdeye.spi.datalayer.dto.AlertTemplateDTO;
+import javax.annotation.Nullable;
 
 public class ResourceIdentifier {
 
   static public final String DefaultNamespace = "default";
```

Comment:
we mostly follow Google coding conventions.

constants should be capitalized and underscored. see Constants.java
```
  static public final String DEFAULT_NAMESPACE = "default";
```